xdsl/dialects/csl/csl.py

```diff
@@ -92,6 +92,8 @@
 from xdsl.utils.isattr import isattr
 from xdsl.utils.str_enum import StrEnum
 
+target = Literal["wse2", "wse3"]
```

Comment:
Nit: could you change this to `Target`, because it's a python type?

Reply:
Thanks, yes. I was contemplating all CAPS but didn't like it.